src/App.tsx

```diff
@@ -1,55 +1,80 @@
-import React from 'react';
+import React, { useMemo, useState, useCallback } from 'react';
 import './App.scss';
+import { debounce } from 'lodash';
```

Comment:
Try to fix the linter error
![image](https://github.com/mate-academy/react_autocomplete/assets/56752562/1d355b09-388a-4d82-ba2f-9f2469598ad1)

Reply:
Done, add package.json, if you mean that.

Reply:
In general, should I also add package-lock.json or package.json is fine.